scripts/fill_in_data.sql

```diff
@@ -0,0 +1,40 @@
+
+INSERT INTO public.products (product_id, title, description, price)
+VALUES('7567ec4b-b10c-48c5-9345-fc73c48a80aa', 'SPLAT Biocalcium Toothpaste', 'Biocalcium is a multiple-action toothpaste with bioactive calcium obtained from eggshell and hydroxyapatite, an essential component in the composition of dental enamel. The high content of these ingredients makes it possible to strengthen and remineralise dental enamel and reduce hypersensitivity.', 5);
+INSERT INTO public.stocks (product_id, count)
+VALUES('7567ec4b-b10c-48c5-9345-fc73c48a80aa', 4);
+
+INSERT INTO public.products (product_id, title, description, price)
+VALUES('7567ec4b-b10c-48c5-9345-fc73c48a80a0', 'PERIOR Pum Ping Toothpaste', 'PERIOE 46cm Pumping Gel Type Toothpaste', 1000);
```

Comment:
You can create one insert for all products instead of each insert for each product